lib/suse_container_urls.pm

```diff
@@ -0,0 +1,77 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Database for URLs of docker images to be tested
+# Maintainer: Fabian Vogt <fvogt@suse.com>
+
+package suse_container_urls;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_sle is_opensuse is_tumbleweed is_leap);
+
+our @EXPORT = qw(
+  get_suse_container_urls
+);
+
+# Returns a tuple of image urls and their matching released "stable" counterpart.
+# If empty, no images available.
+sub get_suse_container_urls {
+    my $version    = get_required_var('VERSION');
+    my $dotversion = $version =~ s/-SP/./r;         # 15 -> 15, 15-SP1 -> 15.1
+    $dotversion = "${dotversion}.0" if $dotversion !~ /\./;    # 15 -> 15.0
+
+    my @image_names  = ();
+    my @stable_names = ();
+    if (is_sle(">=12-SP3")) {
+        my $lowerversion  =~ s/-//r;
```

Comment:
This doesn't actually lowercase the version at all anymore.

Reply:
(Also, travis complains)

Reply:
I don’t follow - what do you want this code to be? You wrote this stuff and I am completely losing your intend as the above code segment is what you wrote.. 

Use the sexy github suggest feature?

Reply:
> Use the sexy github suggest feature?

Doesn't seem to support multiline